docs/source/overview/quicktour.md

```diff
@@ -115,7 +114,9 @@ The engine which handles the auto-tuning process. You can select search strategy
 
 ```json
 "engine": {
-    "cache_dir": ".cache"
+    "host": {"type": "LocalSystem"},
+    "target": {"type": "LocalSystem"},
```

Comment:
>target [](http://example.com/codeflow?start=5&length=6)

it seems the engine need to be changed to host the target?

Reply:
yes, I have opened a PR into this branch to add the other necessary changes. https://github.com/microsoft/Olive/pull/233 

@devang-ml please review, the linked PR and merge it into this branch.